Formula/postgresql.rb

```diff
@@ -94,11 +94,13 @@ def install
   end
 
   def post_install
+    require "open3"
     (var/"log").mkpath
     (var/"postgres").mkpath
     unless File.exist? "#{var}/postgres/PG_VERSION"
       system "#{bin}/initdb", "#{var}/postgres"
     end
+    _stdout, _stderr, _status = Open3.capture3("createdb")
```

Comment:
While I think that should be done, there may be a chance that the `{var}/postgres/PG_VERSION` could exist due to a previous installation and uninstallation (where the database in question is non existant), rendering `unless` ineffective.
I also thought of using another block to test the existence of the db, but couldn't think of one, especially since postgres has its own ways of storing and reading database data. I'd be glad if you could provide a workaround, as even I find using Open3 clumsy, and more of a hack.